.gitignore

```
@@ -1,7 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-/node_modules
+/frontend/node_modules
+/backend/node_modules
```

```suggestion
node_modules
```
このように記述すればすべての階層のnode_modulesフォルダがignoreされます